### PR TITLE
Mynewt test fixes

### PR DESCRIPTION
**Pull request by jrotkiewicz**
Dialog board sends 0 after reset. Added fix to discard this zero.

### CODE REVIEW

**Comment by jrotkiewicz** [comment]
@michal-narajowski

File: wid/gap.py [hunk 1]
```diff
@@ -933,11 +933,24 @@ def hdl_wid_232(desc):
     return True
 
 
+def hdl_wid_400(desc):
+    btp.set_filter_accept_list()
+    bd_addr = '000000000000'
+    bd_addr_type = btp.OwnAddrType.le_resolvable_private_address
```
Comment (michal-narajowski) [inline wid/gap.py:939]: Please update doc/btp_spec.txt:
- Add documentation for Set Filter Accept List
- Update Gap Connect documentation and mention the special behavior when IUT sends zero address.

File: ptsprojects/mynewt/gap_wid.py [hunk 2]
```diff
@@ -104,6 +105,20 @@ def hdl_wid_204(desc):
     return btp.check_discov_results(addr_type=0x02)
 
 
+def hdl_wid_402(desc):
```
Comment (michal-narajowski) [inline ptsprojects/mynewt/gap_wid.py:108]: These handlers are already in wid/gap.py.

File: pybtp/iutctl_common.py [hunk 3]
```diff
@@ -149,6 +149,34 @@ def __init__(self):
         self.event_handler_cb = None
 
     def _rx_task(self):
+
```
Comment (michal-narajowski) [inline pybtp/iutctl_common.py:152]: Please create a separate PR for this. This is Dialog-specific while other changes are independent.

**Review by michal-narajowski — CHANGES_REQUESTED** [review]

**Comment by michal-narajowski** [comment]
Related PR: https://github.com/apache/mynewt-nimble/pull/1018

File: reset.jlink [hunk 4]
```diff
@@ -0,0 +1,2 @@
+r
```
[the remark this answers is not in the document]
Comment (jrotkiewicz) [reply reset.jlink:1]: Not anymore, fixed: https://github.com/intel/auto-pts/pull/639

File: pybtp/iutctl_common.py (hunk 3 above)
Comment (jrotkiewicz) [reply pybtp/iutctl_common.py:152]: https://github.com/intel/auto-pts/pull/639

File: wid/gap.py (hunk 1 above)
Comment (jrotkiewicz) [reply wid/gap.py:939]: https://github.com/intel/auto-pts/pull/640

**Comment by michal-narajowski** [comment]
Please rebase and resolve conflicts

**Comment by michal-narajowski** [comment]
@jrotkiewicz Please rebase and resolve conflicts

**Comment by jrotkiewicz** [comment]
@michal-narajowski 
Necessary nimble fix for [Mynewt : fix GAP/CONN/ACEP/BV-04-C](https://github.com/intel/auto-pts/pull/585/commits/266cd164a3d04c87454c577263f381baa1327c3d): https://github.com/apache/mynewt-nimble/pull/1085

**Review by michal-narajowski — APPROVED** [review]

**Pull request merged**